### PR TITLE
Update changelog for v0.12.0

### PR DESCRIPTION
**Pull request by IodaMikeMapbox**
Checks:

* [x] Update changelog
* [x] Rebase to `release/0.12.0` branch
* [x] Assign reviewers

Linked issues:
mapbox/mapbox-vision#2314

### CODE REVIEW

File: Docs/CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,22 @@
 # Changelog
 
-## 0.12.0 - Unreleased
+## 0.12.0
 
 ### Vision
+- Added detection of Japanese traffic signs
 - Added property `progress` to the `VisionReplayManager`
 - Added read-only property `duration` to the `VisionReplayManager`
+- Added replay support of recorded routes
```
Comment (chezzdev) [inline Docs/CHANGELOG.md:9]: Could you please append this comment to the appropriate line in the AR section?
Like "Added ... method to the ... to support route replay from the recorded session".

File: Docs/CHANGELOG.md [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
 - Changed `VisionReplayManager`'s `start`/`stop` methods functionalities to the `pause`/`resume`
```
Comment (chezzdev) [inline Docs/CHANGELOG.md:10]: Oh, and this line is probably explained in the `VisionReplayManager` behavior one line below. Please remove this one.

File: Docs/CHANGELOG.md [hunk 3]
```diff
@@ -1,16 +1,22 @@
 # Changelog
 
-## 0.12.0 - Unreleased
+## 0.12.0
 
 ### Vision
+- Added detection of Japanese traffic signs
```
Comment (chezzdev) [inline Docs/CHANGELOG.md:6]:
```suggestion
- Added `Japan` country and support for the detection of Japanese traffic signs
```

File: Docs/CHANGELOG.md [hunk 4]
```diff
@@ -1,19 +1,23 @@
...
+- Added `Japan` country and support for the detection of Japanese traffic signs
 - Added property `progress` to the `VisionReplayManager`
 - Added read-only property `duration` to the `VisionReplayManager`
-- Changed `VisionReplayManager`'s `start`/`stop` methods functionalities to the `pause`/`resume`
 - Changed `VisionReplayManager`'s behaviour:
   - method `start` continues session replay from the current progress
   - method `stop` stops session replay without changing the progress
   - end of the session does not trigger `stop` method
 - Deprecated separate detection and segmentation models configuration (use `modelPerformance` instead of `modelPerformanceConfig` on `VisionManager`)
+- Fixed the bug with the inability to set AR visual params
```
Comment (chezzdev) [inline Docs/CHANGELOG.md:14]: Should we move it to the AR section?

Comment (IodaMikeMapbox) [reply Docs/CHANGELOG.md:14]: ah, right

**Pull request merged**